### DISCUSSION TITLE
Running pymagnum in Windows 10

### DISCUSSION THREAD

**Discussion by rmurfster** [CharlesGodwin/pymagnum#43] [General]
I know (almost) nothing about python.  I am a C/C#/Java developer running Windows 10.

I have installed Python 3.10 in Windows 10

Using your manual at "https://pymagnum.readthedocs.io/en/latest/installation.html":
- I installed pymagnum by running "pip install --upgrade pymagnum"
- I ran "python -m serial.tools.list_ports" and that worked.

"pip show pymagnum" shows that pymagnum is installed.
> Name: pymagnum
> Version: 2.0.1
> Summary: Magnum Energy Network Interface (read-only)
> Home-page: https://github.com/CharlesGodwin/pymagnum
> Author: Charles Godwin
> Author-email: magnum@godwin.ca
> License: BSD
> Location: c:\users\richard\appdata\local\packages\pythonsoftwarefoundation.python.3.10_qbz5n2kfra8p0\localcache\local-packages\python310\site-packages
> Requires: pyserial, tzlocal, uptime
> Required-by:

However, running "python magtest --help" gives me the error:
> python: can't open file 'c:\\Users\\richard\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python310\\site-packages\\magnum\\magtest --help': [Errno 2] No such file or directory

Any help would be appreciated.

Thanks!

**Comment by CharlesGodwin** [comment]
I have temporarily lost internet access except my phone. I'll reply on October 29th

**Comment by CharlesGodwin** [comment]
Please forgive me. I failed to return to this.

Jusr run the commands as `magtest --help`, there's no need for the python part.
